services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/landing_page.jsx

```diff
@@ -71,39 +77,52 @@ const contentHubMiniList = (hasAssignedSocialStudiesActivities, hasAssignedScien
     {
       title: 'Social Studies Dashboard',
       href: socialStudiesLink,
-      img: `${window.postbuildEnv.CDN_URL}/images/illustrations/world-history-dashboard.svg`,
+      imgSrc: `${baseImgSrc}/social_studies_dashboard.svg`,
       bodyText: socialStudiesBodyText,
-      flag: null,
       new: true,
       loading
     }, {
       title: 'Interdisciplinary Science Dashboard',
       href: interdisciplinaryScienceLink,
-      img: `${window.postbuildEnv.CDN_URL}/images/illustrations/ai-dashboard.svg`,
+      imgSrc: `${baseImgSrc}/interdisciplinary_science_dashboard.svg`,
       bodyText: interdisciplinaryScienceBodyText,
-      flag: null,
       new: true,
       loading
     }
   ]
 }
 
-const miniBuilder = (mini) => {
-  const newElement = mini.new ? <span className="new-tag">NEW</span> : null
-  const premium = mini.premium ? <h4 className="premium"><span>Premium</span><img alt="" src={`${window.postbuildEnv.CDN_URL}/images/icons/yellow-diamond.svg`} /></h4> : null;
+const Card = ({ isNew, premium, title, id, href, bodyText, imgSrc, loading, }) => {
+  const newElement = isNew ? <span className="gold-new-tag">NEW</span> : null
+  let linkClassName = 'quill-button focus-on-light contained medium '
+  linkClassName += premium ? 'gold' : 'green'
   return (
-    <a className="generic-mini focus-on-light" href={mini.href} id={mini.id} key={mini.title}>
+    <section aria-labelledby={title} className={`report-card ${premium ? 'premium-card' : ''}`} id={id} key={title}>
       {newElement}
-      <h3>{mini.title}</h3>
-      {premium}
-      <div className="img-wrapper">
-        <img alt="" src={mini.img} />
+      <h3 id={title}>{title}</h3>
+      <p>{loading ? <Spinner /> : bodyText}</p>
+      <div className="button-and-img-wrapper">
+        <a className={linkClassName} href={href}>View Report</a>
+        <img alt="" src={imgSrc} />
       </div>
-      <p style={mini.pStyle ? mini.pStyle : {}}>{mini.loading ? <Spinner /> : mini.bodyText}</p>
-    </a>
+    </section>
   );
 };
 
+const ReportSection = ({ cards, imgSrc, title, }) => {
+  return (
+    <section aria-labelledby={title} className="report-section">
+      <div>
+        <img alt="" src={imgSrc} />
```

Comment:
No alt text?

Reply:
No -- I think these images would be considered [purely decorative](https://www.w3.org/WAI/tutorials/images/decorative/) (they don't offer additional context or information to the user), so adding alt text just creates noise. Let me know if you disagree, though!